src/main/java/com/glinka/touk_task/model/TicketType.java

```diff
@@ -0,0 +1,17 @@
+package com.glinka.touk_task.model;
+
+public enum TicketType {
+    ADULT(25),
+    STUDENT(18),
+    CHILD(12.5);
+
+    private double price;
```

Comment:
Czy to jest dobre miejsce do przechowywania ceny biletu?
Jak myślisz, czy taryfa biletowa to coś, co się może zmieniać w kinie?
W jakich innych miejscach można by trzymać takie informacje?